### ISSUE TITLE
PacketFastBreak

### ISSUE THREAD

**Issue by ghost** [Updated-NoCheatPlus/NoCheatPlus#248]
Build 149, config default
Client: https://github.com/bebeli555/CookieClient (opensource,no malware)

https://user-images.githubusercontent.com/108226177/188713417-4722d1ce-2b5e-46e0-96cf-82669d3a0ad4.mp4

**Comment by ghost** [comment]
EF 5 pickaxe

**Comment by Lysandr0** [comment]
This could have been fixed already with the hardness-check overhaul...
@xaw3ep is the hardness branch safe for production? You may want to merge it in master just so we can get some feedback from people, instead of having to test ourselves.

**Comment by Lysandr0** [comment]
Might be fixed with the breaking-time overhaul, although the old leniency settings should be tweaked to match the new speed calculations. 

Edit: Done, remember to regenerate the config file for the new settings to apply. Clients which stay close to vanilla mining speed might get past still, but should be much better than before.

Assume fixed.
